sdk_extensions/zpages/src/main/java/io/opentelemetry/sdk/extensions/zpages/TraceConfigzActiveTableRow.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.sdk.extensions.zpages;
+
+import com.google.auto.value.AutoValue;
+import java.io.PrintStream;
+
+/** Builder pattern class for emiting a single row of the change parameter table. */
+@AutoValue
+abstract class TraceConfigzActiveTableRow {
+  @AutoValue.Builder
+  public abstract static class Builder {
```

Comment:
I'd move the builder definition down below all the abstract methods for the main class. Also, I think Builders can be done with an interface, rather than an abstract class, which makes things read much more cleanly, IMO.

Reply:
Seems like all other classes using `@AutoValue.Builder` are also using `class` for Builder

Reply:
But it can be interface, which will make it cleaner for sure

Reply:
I don't feel super strongly about it...I just discovered it myself last week and like the way it makes the builders feel, as long as you don't need any fancy functionality in your builder class.